Client/Frontend/Browser/TabManager.swift

```diff
@@ -841,34 +832,16 @@ extension TabManager {
         }
     }
 
-    func restoreTabs() {
+    lazy var restoreTabs: () -> Tab = {
```

Comment:
There are two `restoreTabs` now, a lazy var and a function on line 845. I would do some cleaning or renaming for more clarity.
`func restoreTabs` is only used in undoing closing tabs, I would just rename it, or remove it in a separate ticket if we are not going keep restoring closed tabs functionality in 1.7